constraint_generation/src/execution_data/executed_template.rs

```diff
@@ -112,10 +144,27 @@ impl ExecutedTemplate {
     }
 
     pub fn insert_in_dag(&mut self, dag: &mut DAG) {
```

Comment:
The information computed when executing the template regarding the order of signals and "constraints" generated when substituting a custom gate's signal is sent to the corresponding node in the DAG to be used later.